.travis.yml

```diff
@@ -20,8 +20,6 @@ matrix:
     env: OCAML_VERSION=4.02 OPAM_VERSION=1.1.2
   - os: osx
     env: OCAML_VERSION=4.02 OPAM_VERSION=1.2.2
-  allow_failures:
-  - env: OCAML_VERSION=4.03 OPAM_VERSION=1.2.2
```

Comment:
4.03 is [still in the test matrix](https://github.com/yallop/opam-repository/blob/673fa190743c25743dd2a5a382e7ce7ed9f8e686/.travis.yml#L15-L16).  It's no longer allowed to fail.